### PR TITLE
Pass watch option along to website template.

### LINKED ISSUE #241: Live reload / watch not working since 2.4.1
I upgraded to the 3.0 series from 2.4.1 and the live reload option stopped working.

### PR DESCRIPTION
**Pull request by HoverBaum**
Should fix #241.

### CODE REVIEW

**Comment by HoverBaum** [comment]
Feels good to once again work on this feature, after a two year break 🙂

File: lib/config.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ module.exports.getAssetsDir = () => mergedConfig.assetsDir;
 module.exports.getStaticDir = () => (mergedConfig.static === true ? mergedConfig.staticDir : mergedConfig.static);
 module.exports.getHost = () => mergedConfig.host;
 module.exports.getPort = () => mergedConfig.port;
-module.exports.getWatch = () => Boolean(mergedConfig.watch);
+module.exports.getWatch = () => Boolean(mergedConfig.watch || mergedConfig.w);
```
Comment (webpro) [inline lib/config.js]: Thanks for the PR, @HoverBaum! However, I think this should be fixed somewhere here: https://github.com/webpro/reveal-md/blob/master/bin/reveal-md.js#L13-L22

Comment (HoverBaum) [reply lib/config.js]: That sounds reasonable. The library sued should probably handle setting `watch` for both `-w` and `--watch`.

Will tackle this angle tomorrow.

**Comment by HoverBaum** [comment]
@webpro added a new commit moving the `w: 'watch'` alias to another config parsing from where we can actually hand it on to the template. That will hopefully be a clean fix to the watch issue.

**Comment by webpro** [comment]
Thanks @HoverBaum! Released in [v3.0.4](https://github.com/webpro/reveal-md/releases/tag/3.0.4)

**Pull request merged**